bindings/python/cntk/ops/__init__.py

```diff
@@ -261,28 +261,87 @@ def labels_to_graph(labels, name=''):
     return labels_to_graph(labels, name)
 
 @typemap
-def forward_backward(graph, features, blankTokenId, delayConstraint=-1, name=''):
+def forward_backward(labels_graph, network_out, blank_token_id,
+                     delay_constraint=-1, name=''):
     '''
-    Criterion node for training methods that rely on forward-backward Viterbi-like passes, e.g. Connectionist Temporal Classification (CTC) training
-    The node takes as the input the graph of labels, produced by the labels_to_graph operation that determines the exact forward/backward procedure.
+    Criterion node for training methods that rely on forward-backward Viterbi-like
+    passes, e.g. Connectionist Temporal Classification (CTC) training.
+    The node takes as the input the graph of labels, produced by the labels_to_graph
+    operation that determines the exact forward/backward procedure.
+
     Example:
-        graph = cntk.labels_to_graph(labels)
-        networkOut = model(features)
-        fb = C.forward_backward(graph, networkOut, 132)
+        labels_graph = cntk.labels_to_graph(labels_var)
+        network_out = model(input_var)
+        fb = C.forward_backward(labels_graph, network_out, 132)
+
+    Hint:
+        CNTK requires that both features and labels have the same time dimension.
+        This forward-backward code expects the duplication of these labels to
+        fulfill the requirements. The labels must be encoded as one-hot vectors,
+        where the positive index has values 1 or 2 (from now on referred to as
```

Comment:
Did you mean "populated" instead of "positive"?

Reply:
Yes, you are right!